packages/domains-table/src/utils/url-support.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line wpcalypso/i18n-unlocalized-url
+const root = 'https://wordpress.com/support';
```

Comment:
An option for avoiding this warning would be use the `useLocalizeUrl` hook (also from `@automattic/i18n-utils`) and creating some sort of `useSupportUrls()` hook that could return the localised urls.